tox.ini

```diff
@@ -19,12 +20,14 @@ deps =
        -rrequirements/requirements-optionals.txt
 
 [testenv:py27-flake8]
+downloadcache={env:PIP_DOWNLOAD_CACHE}
 deps =
        -rrequirements/requirements-testing.txt
        -rrequirements/requirements-codestyle.txt
 commands = ./runtests.py --lintonly
 
 [testenv:py27-docs]
+downloadcache={env:PIP_DOWNLOAD_CACHE}
```

Comment:
@maryokhin 

You're right about the ``downloadcache`` inheritance and ``downloadcache``. I'm redoing this to use wheels instead.